lib/atol/request/post_document/item/body.rb

```diff
@@ -21,19 +22,35 @@ class Body
             'composite', 'another'
           ]
 
-          attr_accessor :config, :name, :price, :quantity, :payment_method, :payment_object
+          AGENT_TYPES = [
+            'bank_paying_agent', 'bank_paying_subagent', 'paying_agent',
+            'paying_subagent', 'attorney', 'commission_agent', 'another'
+          ]
+
+          attr_accessor :config, :name, :price, :quantity, :payment_method,
+                        :payment_object, :agent_type, :supplier_phones, :supplier_name,
+                        :supplier_inn
 
-          def initialize(config: nil, name:, price:, quantity: 1, payment_method:, payment_object:)
+          def initialize(
+            config: nil, name:, price:, quantity: 1, payment_method:,
+            payment_object:, agent_type: nil, supplier_phones: nil,
+            supplier_name: nil, supplier_inn: nil
+          )
             raise Atol::ZeroItemQuantityError if quantity.to_f.zero?
             raise BadPaymentMethodError unless PAYMENT_METHODS.include?(payment_method.to_s)
             raise BadPaymentObjectError unless PAYMENT_OBJECTS.include?(payment_object.to_s)
+            raise BadAgentTypeError unless agent_type.nil? || AGENT_TYPES.include?(agent_type.to_s)
 
             self.config = config || Atol.config
             self.name = name
             self.price = price.to_f
             self.quantity = quantity.to_f
             self.payment_method = payment_method.to_s
             self.payment_object = payment_object.to_s
+            self.agent_type = agent_type
```

Comment:
Я бы предложил данное значение также приводить к строке, раз уж это делается для `payment_method` и `payment_object`

```suggestion
            self.agent_type = agent_type.to_s
```